types/airbnb-prop-types/index.d.ts

```diff
@@ -24,6 +24,12 @@ export type ReactFunctionComponentLike = (...args: any[]) => PropTypes.ReactNode
 
 export type ReactTypeLike = string | ReactClassComponentLike | ReactFunctionComponentLike;
 
+export interface ReactRefLike<T> {
+    readonly current: T | null;
+}
+
+export type ReactLegacyRefLike<T> = (instance: T | null) => void | ReactRefLike<T> | null;
```

Comment:
Not totally sure we want `| null` or not. Included it because React's definition for ref does.

Reply:
Meh, decided to remove it